frontends/ol-components/src/components/SearchInput/SearchInput.tsx

```diff
@@ -15,6 +18,29 @@ export interface SearchSubmissionEvent {
   preventDefault: () => void
 }
 
+const StyledInput = styled(Input)`
+  border-top-right-radius: 0;
+  border-bottom-right-radius: 0;
+  max-width: 81%;
+  border-right: none;
+  height: 37px;
+`
+
+const StyledButton = styled(Button)`
```

Comment:
The mobile view swaps out the search button for an icon inside the input, https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=4026-58868&m=dev

<img width="571" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/b139ef2a-6415-456c-90ed-7d41772e444e">

Reply:
Also adds 8px vertical padding to make taller

<img width="571" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/5a6c22df-6dcc-4cd7-bb0c-990e2e8b97a9">

Reply:
That's an old design that hasn't been update on mobile. The mobile view is supposed to have the same search bar as desktop. The tabs in Figma are also not updated to the new design

Reply:
@jonkafton @abeglova  Updating mobile UI in a moment

Reply:
UI updated, Thanks for pointing it out 👍 

![image](https://github.com/mitodl/mit-open/assets/7486671/22a0d975-7aa9-4aed-9f04-281424a03fbc)

also @abeglova Please check the spacing of the [filter, clear all, & Close ](https://www.figma.com/design/Eux3guSenAFVvNHGi1Y9Wm/MIT-Design-System?node-id=4026-61843&t=Maq5E3Q8L8jxS6gd-1) along with the things we have earlier today 👍

Reply:
Thanks @mbilalmughal. Update received 

<img width="412" alt="image" src="https://github.com/mitodl/mit-open/assets/939376/ee3ab7a5-ec8d-4e9e-a98e-43b2ba8929b7">